### ISSUE TITLE
Add more configuration examples

### ISSUE THREAD

**Issue by pattrickrice** [wayou/vscode-todo-highlight#172]
Maybe a directory of example configurations? I attempted to tweak the settings and found it surprisingly difficult.

**Comment by NikolaiVsevolodovich** [comment]
Yeah I started out writing some regex tags and all but then I realized that no matter what they were all the default color?

Ive played around with my keywords and as far as I can tell in terms of styling youre really limited to font weight/style and outline. I found border cramped the text too much, outline gives it some breathing room, but then it eats into the lines above and below so ymmv. It would be nice if vscode allowed us to have individual line fontsize/family. Ive got isWholeLine set to true and It would also be cool if the lines could "blend" into each other.. Im pretty bad for my todos or fixit explanations to eat up a couple lines easy and its not ideal

I also initially thought that somehow the text would be excluded from being parsed, but discovered that I have to put inline comment chars  for every line..


Ive still got a lot of playing around to do..

I came here trying to find out how I can delete the stock keywords/colors.. now where was I...

**Comment by WybeBosch** [comment]
If you are looking what options you have to Style i found this VScode documentation that shows you the options this extension can use: https://code.visualstudio.com/api/references/vscode-api

so there is more than just "backgroundColor" etc. If you want to get funky you can also use the "before" to add spacing / text.  e.g.: 

`
"before" : {
	"textContent" :"henk"
}
`

Also if you if you want to be able to have Individual regex matches, i suggest checking out: #152

**Comment by jgclark** [comment]
I fully agree @pattrickrice and @GTMxCode ! I've been helped by digging into the PR that @vonEdfra did for #152, but still too much is unclear. For example, I'm not sure why my settings file:
```json
"todohighlight.keywords": [
		"TODO:",
		"FIXME:",
		"@@@",
		"FINISH:",
		{
			"text": "TODO",
			"color": "white",
			"backgroundColor": "#ffbd6a",
			"overviewRulerColor": "rgba(255,189,42,0.8)"
		}, etc.
```
shows three *different* colour patterns for what should be, I think, the same pattern.
It feels like the built-in `TODO:` and `FIXME:` settings are showing through and not being overridden as I hoped I was doing.

**Comment by jgclark** [comment]
> If you are looking what options you have to Style i found this VScode documentation that shows you the options this extension can use: https://code.visualstudio.com/api/references/vscode-api

Thanks, @WybeBosch. I was interested, so went to this (very long) page. I found lots and lots on color options, but nothing else. There's almost no mention of CSS properties in the rest of the documentation. Can you point me more specifically to this?
 
> so there is more than just "backgroundColor" etc. If you want to get funky you can also use the "before" to add spacing / text. e.g.:
> 
> `"before" : { "textContent" :"henk" }`

I tried this, and it isn't working for me. https://cssreference.io/property/content/ suggests that it might want to be "content" not "textContent", and will need to be some variant of ":before" not "before"?

**Comment by jgclark** [comment]
OK, I've now worked more of this out. In the previous example, the FIXME:, @@@ and FINISH: all have no further definition, and so just pick up whatever is in set in the `defaultStyle`. It's only the TODO: that's freshly defined here.

**Comment by jgclark** [comment]
I've now forked this extension into https://github.com/jgclark/vscode-todo-highlight, and made available on the Marketplace. It includes better documentation, more examples, update for remote development, addition of regex per highlight etc. 

You'll need to uninstall this extension and then install the new one from the marketplace, [TODO Highlight v1.2.0](https://marketplace.visualstudio.com/items?itemName=jgclark.vscode-todo-highlight).
